app/Http/Controllers/API/CustomTourController.php

```diff
@@ -10,9 +10,22 @@ class CustomTourController extends BaseController
 {
     public function store(Request $request)
     {
-        $data = $request->json()->all();
-
-        $record = CustomTour::create(['tour_json' => json_encode($data)]);
+        $data = $request->validate(
+            [
```

Comment:
Move these to `app/Http/Requests/API/CustomTourRequest`

Reply:
Thanks, I've now moved the validation rules to `app/Http/Requests/API/CustomTourRequest`